i18n-guidelines.md

```diff
@@ -1,24 +1,96 @@
+
+1. **Identify target languages**.
+
+   Determine which languages you want to add translations for. Make sure these languages are relevant to the project's user base.
+
+2. **Create translation files**.
+
+   Inside the "translations" directory, create a new subdirectory for each language you plan to support. Use language codes (e.g., "en" for English, "fr" for French) as directory names.
+
+   ```markdown
+   .
+   └── translations/
+      ├── en/ (English)
+      ├── fr/ (French)
+      └── es/ (Spanish)
+   ```
+
+3. **Translate content**.
+
+   - For each language directory, create translated versions of the documentation files. Typically, you translate Markdown files but consider other formats as needed.
+   - Maintain the same file names and structure as in the original documentation but with translated content.
+
+4. **Update links**.
+
+   In the translated files, ensure that any internal links (e.g., links to other sections or pages within the documentation) are updated to point to the corresponding translated content.
+
+5. **Add language selector**.
+
+   Consider adding a language selector to the documentation, allowing users to switch between different language versions. You can do this by modifying the navigation or header of your documentation.
+
+6. **Testing and validation**.
+
+   Test the translated documentation to ensure accuracy and readability. Ensure all links work correctly and the content is culturally appropriate.
+
+7. **Submit translations**.
+
+   If you haven't already, submit your translations as a PR. Ensure you provide clear information about the languages you've translated and any specific details related to your contributions.
+
+8. **Collaborate and review**.
+
+   Collaborate with other contributors and reviewers to ensure the quality of translations. Be open to feedback and suggestions for improvement.
+
+### Best Practices for Translations
+
+- Maintain consistency in terminology and style throughout the translated documentation.
+- Work with another contributor who speaks the language you're translating to.
+- It helps to mention specific tools you use so developers who want to translate documentation can see how it's done.
+- Keep translations up to date with changes in the original documentation.
 
 ## Reviewer Process Guide
 
-When it comes to reviewing a translation pull request (PR), ask yourself the following questions:
+When it comes to reviewing a translation PR, ask yourself the following questions:
 
-* Does the current translation match the instructions in the English version?
-* Are there links that could be localized? (e.g. Wikipedia and MDN links)
-* Is the translation correctly written following the translated language's norms and practices?
+- Does the current translation match the instructions in the English version?
+- Are there links that could be localized? (e.g., Wikipedia and MDN links)
```

Comment:
Links that could be localized for what purpose?

Reply:
To see if it has a translation option mostly.

Reply:
Hmmm... Sorry. I still don't get it 😅

Reply:
It means if the other source can be translated too @adiati98 😁

Reply:
Ah, okay.

Should we say:

```suggestion
- Are there links that could be localized and translated? (e.g., Wikipedia and MDN links)
```